<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryInspectorDockWidget</class>
 <widget class="QDockWidget" name="MemoryInspectorDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionBreak_on_CPU_access_here">
   <property name="text">
    <string>Break on CPU access here</string>
   </property>
   <property name="toolTip">
    <string>Break on CPU access here</string>
   </property>
  </action>
  <action name="actionBreak_on_CPU_read_here">
   <property name="text">
    <string>Break on CPU read here</string>
   </property>
   <property name="toolTip">
    <string>Break on CPU read here</string>
   </property>
  </action>
  <action name="actionBreak_on_CPU_write_here">
   <property name="text">
    <string>Break on CPU write here</string>
   </property>
   <property name="toolTip">
    <string>Break on CPU write here</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
